src/main/java/org/numenta/nupic/Parameters.java

```diff
@@ -97,7 +97,11 @@
         //////////// Spatial Pooler Parameters ///////////
         Map<KEY, Object> defaultSpatialParams = new ParametersMap();
         defaultSpatialParams.put(KEY.INPUT_DIMENSIONS, new int[]{64});
-        defaultSpatialParams.put(KEY.POTENTIAL_RADIUS, 16);
+        /** <b>WARNING:</b> potentialRadius **must** be set to 
+         * the inputWidth if using "globalInhibition" and if not 
+         * using the Network API (which sets this automatically) 
+         */
+        defaultSpatialParams.put(KEY.POTENTIAL_RADIUS, -1);
```

Comment:
I removed this one, but I added it (where it should have been), on the key itself (where it will be looked up).